docs/user/reference/general/initializing.md

```diff
@@ -178,6 +218,16 @@ Glean.initialize(
 ```
 </div>
 
+<div data-lang="Firefox Desktop" class="tab">
+
+{{#include ../../../shared/blockquote-warning.html}}
+
+##### No custom initialization needed
+
+> On Firefox Desktop Glean initialization is handled internally.
```

Comment:
nit: This information could be presented as a data-info tooltip instead of requiring the tab to be opened :)